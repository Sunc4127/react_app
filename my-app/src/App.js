import "./App.css";
import "./components/dogs.css";
import Dogs from "./components/dogs";

function App() {
    return (
        <main className="container">
            <Dogs />
        </main>
    );
}

export default App;
