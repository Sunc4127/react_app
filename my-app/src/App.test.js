
import App from './App';
import React from "react";
import ReactDOM from "react-dom";
import { render, screen } from "@testing-library/react";
import Dog from './components/dogs.jsx';

it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<App />, div);
});



it("renders loading language", () => {
  render(<App />);
  expect(
    screen.getByText("Images and Videos Loading......")
  ).toBeInTheDocument();
});

test("Check the length of the list", () => {
  let dog = new Dog();
  return dog.componentDidMount().then(() => {
      expect(dog.state.dogs.length).toBe(8);
  });
});
