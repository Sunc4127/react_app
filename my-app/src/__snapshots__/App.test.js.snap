// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<link
  page="http://www.facebook.com"
>
  Facebook
</link>
`;
